### PR TITLE
new `low_speed_limit` and `low_speed_time`  options

### PR DESCRIPTION
**Pull request by manuelbb-upb**
This is what I came up with to circumvent issue https://github.com/JuliaComputing/gRPCClient.jl/issues/22
Both options default to 0 in accordance with the corresponding Curl options.

### CODE REVIEW

**Comment by manuelbb-upb** [comment]
The second commit reverts my change to the version number in Project.toml. I used that for testing purposes, but of course versioning should be up to the project maintainer :)

**Comment by manuelbb-upb** [comment]
This is relevant here also: https://github.com/JuliaLang/Downloads.jl/issues/193

File: src/curl.jl [hunk 1]
```diff
@@ -160,15 +160,29 @@ function recv_data(easy::Curl.Easy, output::Channel{T}, max_recv_message_length:
...
 function set_connect_timeout(easy::Curl.Easy, timeout::Real)
     timeout >= 0 ||
         throw(ArgumentError("timeout must be positive, got $timeout"))
     if timeout ≤ typemax(Clong) ÷ 1000
         timeout_ms = round(Clong, timeout * 1000)
         Curl.setopt(easy, CURLOPT_CONNECTTIMEOUT_MS, timeout_ms)
     else
-        timeout = timeout ≤ typemax(Clong) ? round(Clong, timeout) : Clong(0)
-        Curl.setopt(easy, CURLOPT_CONNECTTIMEOUT, timeout)
+        Curl.setopt(easy, CURLOPT_CONNECTTIMEOUT, Clong(0))
```
Comment (tanmaykm) [inline src/curl.jl:185]: This would disregard the specified timeout value. Was this intentional?

**Comment by tanmaykm** [comment]
@manuelbb-upb are there any more changes to be done here? If not, maybe mark it ready?

**Comment by tanmaykm** [comment]
Closing, as these are most likely not needed. Ref: https://github.com/JuliaComputing/gRPCClient.jl/issues/22#issuecomment-1189813999

**Pull request closed without merging**